app/models/va_profile_redis/cache.rb

```diff
@@ -16,5 +16,19 @@ def self.invalidate(user)
 
       contact_info.destroy if contact_info.present?
     end
+
+    # @return [Boolean] true if user is a title 38 veteran
+    def veteran?
```

Comment:
Should this be on the base class here? Seems like it belongs on the va_profile_redis/veteran_status class (which I do see there too)

Reply:
Specs are still passing with this method - removed!